compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java

```diff
@@ -365,7 +365,7 @@
                     ParserRuleContext.MINUS_TOKEN, ParserRuleContext.NIL_LITERAL };
 
     private static final ParserRuleContext[] LIST_CONSTRUCTOR_RHS =
-            { ParserRuleContext.CLOSE_BRACKET, ParserRuleContext.EXPRESSION };
+            { ParserRuleContext.CLOSE_BRACKET, ParserRuleContext.EXPRESSION, ParserRuleContext.ELLIPSIS };
```

Comment:
This only works for the first member. It directly calls `EXPRESSION` for the rest of the members. https://github.com/ballerina-platform/ballerina-lang/blob/c30d2bd5bd58d3cc0688593c88b6b041dbcf250c/compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/BallerinaParserErrorHandler.java#L4334-L4336

Also the existing method an naming looks inefficient shall we check an improve it with this as well